controllers/volumereplicationgroup_controller.go

```diff
@@ -1146,6 +1148,43 @@ func (v *VRGInstance) updateVRGLastGroupSyncTime() {
 	v.instance.Status.LastGroupSyncTime = leastLastSyncTime
 }
 
+func (v *VRGInstance) updateVRGLastGroupSyncDuration() {
+	var maxLastSyncDuration *metav1.Duration
+
+	for _, protectedPVC := range v.instance.Status.ProtectedPVCs {
+		if maxLastSyncDuration == nil {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+
+			continue
+		}
+
+		if protectedPVC.LastSyncDuration != nil &&
+			protectedPVC.LastSyncDuration.Duration > maxLastSyncDuration.Duration {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+		}
+	}
+
+	v.instance.Status.LastGroupSyncDuration = maxLastSyncDuration
+}
+
+func (v *VRGInstance) updateLastGroupSyncBytes() {
+	var totalLastSyncBytes *int64
+
+	for _, protectedPVC := range v.instance.Status.ProtectedPVCs {
+		if totalLastSyncBytes == nil {
+			totalLastSyncBytes = protectedPVC.LastSyncBytes
```

Comment:
Same as the comment above. You are assigning a pointer to a pointer. Both points to the same memory location

Reply:
I have tested this PR. I was able to get the values.